mayastor-test/e2e/common/util.go

```diff
@@ -45,13 +45,13 @@ func LabelNode(nodename string, label string) {
 }
 
 // Status part of the mayastor volume CRD
-type mayastorVolStatus struct {
-	state  string
+type MayastorVolStatus struct {
+	State  string
```

Comment:
Nit - what is the reason for capitilising "State" but not the other members? It just looks a little inconsistent.

Reply:
Fixed, all capitalised for export reasons.